stdlib/public/core/Builtin.swift

```diff
@@ -658,6 +658,12 @@ internal func _isUnique<T>(_ object: inout T) -> Bool {
   return Bool(Builtin.isUnique(&object))
 }
 
+@_silgen_name("swift_hasSideTable")
```

Comment:
@mikeash / @gparker42 do we want to keep the 'has side table' bits or should I just remove them?